--- a/rules/base.xml.in_old	2017-05-07 20:14:16.534621625 +0300
+++ b/rules/base.xml.in	2017-05-07 20:15:30.247246730 +0300
@@ -4070,16 +4070,25 @@
             <name>winkeys</name>
             <_description>Romanian (WinKeys)</_description>
           </configItem>
         </variant>
       </variantList>
     </layout>
     <layout>
       <configItem>
+        <name>chm</name>
+        <shortDescription>chm</shortDescription>
+        <description>Mari</description>
+        <languageList><iso639Id>chm</iso639Id></languageList>
+      </configItem>
+      <variantList/>
+    </layout>
+    <layout>
+      <configItem>
         <name>ru</name>
         <!-- Keyboard indicator for Russian layouts -->
         <_shortDescription>ru</_shortDescription>
         <_description>Russian</_description>
         <languageList>
           <iso639Id>rus</iso639Id>
         </languageList>
       </configItem>
@@ -4223,25 +4232,16 @@
             <_description>Bashkirian</_description>
             <languageList>
               <iso639Id>bak</iso639Id>
             </languageList>
           </configItem>
         </variant>
         <variant>
           <configItem>
-            <name>chm</name>
-            <_description>Mari</_description>
-            <languageList>
-              <iso639Id>chm</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
             <name>phonetic_azerty</name>
             <_description>Russian (phonetic azerty)</_description>
           </configItem>
         </variant>
         <variant>
           <configItem>
             <name>phonetic_dvorak</name>
             <_description>Russian (phonetic dvorak)</_description>

